FileUpload/package.json

```diff
@@ -1,40 +1,54 @@
 {
   "name": "dnn-file-upload",
-  "version": "0.1.12",
+  "version": "0.1.13",
   "description": "DNN File Uploader",
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "build": "set NODE_ENV=production&&webpack -p"
+    "eslint": "eslint ./src/**/*.jsx",
+    "prepublishOnly": "npm-run-all prepublishOnly:*",
+    "prepublishOnly:eslint": "eslint ./src/**/*.jsx",
+    "prepublishOnly:babel": "babel src -d lib --source-maps",
+    "prepublishOnly:copy-less": "cpy **/*.less ../lib/ --cwd=src --parents",
+    "prepublishOnly:copy-svg": "cpy **/*.svg ../lib/ --cwd=src --parents"
   },
   "license": "MIT",
   "repository": {
     "type": "git",
     "url": "https://github.com/dnnsoftware/Dnn.Evoq.Microservices.UI.Common.git"
   },
+  "babel": {
+    "presets": [
+      "es2015",
+      "react"
+    ],
+    "plugins": [
+      "transform-object-assign",
+      "transform-object-rest-spread"
+    ]
+  },
   "devDependencies": {
-    "babel-core": "6.24.1",
-    "babel-loader": "6.2.4",
+    "babel-cli": "^6.18.0",
+    "babel-core": "6.24.0",
+    "babel-eslint": "^8.0.1",
+    "babel-plugin-transform-object-assign": "^6.22.0",
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
     "babel-preset-es2015": "6.6.0",
     "babel-preset-react": "6.5.0",
-    "babel-register": "6.16.3",
-    "css-loader": "0.23.1",
-    "dnn-global-styles": "0.0.5",
+    "cpy-cli": "^1.0.1",
     "eslint": "2.10.2",
-    "eslint-config-dnn": "*",
-    "eslint-import-resolver-webpack": "^0.3.2",
-    "eslint-loader": "1.3.0",
-    "eslint-plugin-filenames": "^1.0.0",
-    "eslint-plugin-import": "^1.10.2",
-    "eslint-plugin-react": "5.1.1",
-    "eslint-plugin-spellcheck": "0.0.7",
-    "less": "2.7.1",
-    "less-loader": "2.2.3",
+    "eslint-config-dnn": "^0.0.7",
+    "eslint-import-resolver-node": "^0.3.1",
+    "eslint-plugin-babel": "^4.1.2",
+    "eslint-plugin-import": "^2.8.0",
+    "npm-run-all": "^4.1.1"
+  },
+  "dependencies": {
+    "dnn-global-styles": "0.0.5",
     "moment": "2.13.0",
     "raw-loader": "0.5.1",
-    "react-custom-scrollbars": "4.0.0-beta.1",
-    "style-loader": "0.13.1",
-    "url-loader": "0.5.7",
-    "webpack": "1.13.0"
+    "react": "^15.6.2",
+    "react-custom-scrollbars": "^4.1.2",
```

Comment:
Hm, are we using more than one solution for scrollbars?

Reply:
yes, I also found different scrollbars' libraries in other projects.